autoscoper/src/net/Socket.h

```diff
@@ -48,6 +48,8 @@
 #include <QObject>
 #include "ui/AutoscoperMainWindow.h"
 
+#define SERVER_VERSION "1.0"
```

Comment:
Consider adding the following static functions to the `Socket` object.

In `Socket.h`:

```cpp
/// @{
/// The Autoscoper Socket major/minor/patch version.
///
/// \see https://semver.org/spec/v2.0.0.html for details.
int constexpr versionMajor();
int constexpr versionMinor();
int constexpr versionPatch();
/// @}

/// The Autoscoper Socket version as a string, in MAJOR.MINOR.PATCH format.
QString versionString();
```

In `Socket.cpp`:

```cpp
#define AUTOSCOPER_SOCKET_VERSION_MAJOR 1
#define AUTOSCOPER_SOCKET_VERSION_MINOR 0
#define AUTOSCOPER_SOCKET_VERSION_MINOR 0

int constexpr Socket::versionMajor() { return AUTOSCOPER_SOCKET_VERSION_MAJOR; }
int constexpr Socket::versionMinor() { return AUTOSCOPER_SOCKET_VERSION_MINOR; }
int constexpr Socket::versionPatch() { return AUTOSCOPER_SOCKET_VERSION_PATCH; }

QString Socket::versionString() {
  return QString("v%1.%2.%3").arg(Socket::versionMajor(), Socket::versionMinor(), Socket::versionPatch());
}
```